src/config_json.rs

```diff
@@ -68,6 +68,18 @@ fn get_device_type(config_json: &ConfigMap) -> Result<Option<String>> {
     }
 }
 
+pub fn get_root_certificate(config_json: &ConfigMap) -> Result<Option<String>> {
+    if let Some(value) = config_json.get("balenaRootCA") {
```

Comment:
Note that `balenaRootCA` is in base64-encoded PEM format. Seeing the code, it seems you expect straight PEM format.

Reply:
Thanks, I will add base64 decoding. I saw now how it is decoded for the supervisor: https://github.com/balena-os/meta-balena/blob/bbeae13aee68795e33f9913ad9b430215841211e/meta-resin-common/recipes-containers/resin-supervisor/resin-supervisor/start-resin-supervisor#L11